build.tmpl.yml

```diff
@@ -364,6 +364,16 @@ dockers:
     ids:
       - alpine-cgo-amd64
     build_flag_templates:
+      - "--label=org.opencontainers.image.title={{ .ProjectName }}"
+      - "--label=org.opencontainers.image.description=Scalable, security first OAuth 2.0 and OpenID Connect server"
+
+      - "--label=org.opencontainers.image.vendor=Ory"
+      - "--label=org.opencontainers.image.version={{ .Version }}"
+      - "--label=org.opencontainers.image.created={{ .Date }}"
+      - "--label=org.opencontainers.image.source=https://github.com/ory/hydra"
+      - "--label=org.opencontainers.image.revision={{ .FullCommit }}"
+      - "--label=org.opencontainers.image.url=https://www.ory.sh/hydra"
+      - "--label=org.opencontainers.image.documentation=https://www.ory.sh/docs/hydra"
```

Comment:
Is there a way to template the URLs and descriptions? This config is used for all of our projects

Reply:
I have no clue. Maybe with [custom variable ?](https://goreleaser.com/customization/templates/#custom-variables)
I know it can be done also within a Dockerfile, see for instance what we have done on Traefik Image: https://github.com/traefik/traefik-library-image/pull/33

Reply:
@aeneasr I found a `GitUrl` variable. I updated the PR to make it multi-project compatible. Wdyt ?

Reply:
@aeneasr I updated the PR to make it multi-project compatible. Wdyt ?